src/netius/extra/proxy_c.py

```
@@ -0,0 +1,283 @@
+            self.consul_url
+            + "/v1/health/service/"
+            + netius.legacy.quote(service)
+            + "?passing=true"
+        )
+        result = self._consul_get(url)
+        if result == None:
+            return []
+        return result
+
+    def _consul_get(self, url):
+        headers = dict()
```

Pylint (4.0.4)

[refactor] Consider using '{}' instead of a call to 'dict'.

(R1735)